internal/converter/types.go

```diff
@@ -133,6 +132,22 @@ func (c *Converter) convertField(curPkg *ProtoPackage, desc *descriptor.FieldDes
 			jsonSchemaType.Type = gojsonschema.TYPE_STRING
 		}
 
+	case descriptor.FieldDescriptorProto_TYPE_BYTES:
+		if c.AllowNullValues {
+			jsonSchemaType.OneOf = []*jsonschema.Type{
+				{Type: gojsonschema.TYPE_NULL},
+				{
+					Type:           gojsonschema.TYPE_STRING,
+					Format:         "binary",
+					BinaryEncoding: "base64",
+				},
+			}
+		} else {
+			jsonSchemaType.Type = gojsonschema.TYPE_STRING
+			jsonSchemaType.Format = "binary"
```

Comment:
The JSONSchema library i'm using doesn't seem to know about content encoding

Reply:
Yeah, it doesn't seem to.

It might be related to not supporting the latest schema version.

https://github.com/alecthomas/jsonschema/issues/48

---

I think this is fine / an improvement.